test/test_files/issue/issue3.test

```diff
@@ -1,8 +1,2075 @@
 -GROUP IssueTest
--DATASET CSV empty
+-DATASET CSV EMPTY
```

Comment:
can you add another test which returns every property of the ldbc comment/person table?
I remembered that it caused an exception before.